src/cupbearer/scripts/eval_classifier.py

```diff
@@ -22,37 +19,20 @@ def main(cfg: Config):
         dataset,
         batch_size=cfg.max_batch_size,
         shuffle=False,
-        collate_fn=numpy_collate,
     )
 
-    # Dataloader returns images and labels, only images get passed to model
-    images, _ = next(iter(dataloader))
-    example_input = images[0:1]
-
-    model = cfg.model.build_model()
-    params = cfg.model.build_params()
-    if params:
-        override_variables = {"params": params}
-    else:
-        override_variables = None
-
-    trainer = ClassificationTrainer(
-        num_classes=cfg.num_classes,
-        model=model,
-        optimizer=None,
-        example_input=example_input,
-        log_dir=None,
-        loggers=[],
-        enable_progress_bar=cfg.pbar,
-        rng=jax.random.PRNGKey(cfg.seed),
-        override_variables=override_variables,
+    assert cfg.dir.path is not None
+    classifier = Classifier.load_from_checkpoint(
+        cfg.dir.path / "model.ckpt", test_loader_names=["test"]
```

Comment:
```suggestion
        cfg.dir.path / "last.ckpt", test_loader_names=["test"]
```
This should be last, no?